<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lanplay Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="server_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>27</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string comment="onlineColumn">Online</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string comment="idleColumn">Idle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string comment="serveraddressColumn">Server Address</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="launch_server_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_server_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_server_button">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_list_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>101</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh List</string>
    </property>
   </widget>
   <zorder>server_list</zorder>
   <zorder>add_server_button</zorder>
   <zorder>delete_server_button</zorder>
   <zorder>refresh_list_button</zorder>
   <zorder>launch_server_button</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
